hal/lib/shared/FRCDriverStation.cpp

```diff
@@ -175,4 +231,17 @@ void HAL_ObserveUserProgramTest(void) {
   FRC_NetworkCommunication_observeUserProgramTest();
 }
 
+/**
+ * Waits for the newest DS packet to arrive. Note that this is a blocking call.
+ */
+void HAL_WaitForDSData() {
```

Comment:
Because I just copied it from the old file, and didn't add it into this one. I'll fix it after the build finishes.